### DISCUSSION TITLE
Custom URL-s, getting IP address without prefix

### DISCUSSION THREAD

**Discussion by kotso** [netbox-community/netbox#8701] [Q&A] [answered]
Hello,

Trying to create custom link for IP address, but system that I want to link to, needs only ip, without prefix.

I checked in here: https://github.com/netbox-community/netbox/blob/master/netbox/ipam/models/ip.py and did not find it (not sure its not there, maybe my python skills are that bad)

so how I can get it?

**Comment by candlerb** [comment, marked as answer]
Try `{{ object.address.ip }}`

Here's an example I have that works on Device rather than IPAddress:

### Link Text
```
{% if obj.interfaces.filter(mgmt_only=True).count() == 1 and
   obj.interfaces.get(mgmt_only=True).ip_addresses.count() == 1 %}Open IPMI{% endif %}
```

### Link URL
```
https://{{ obj.interfaces.get(mgmt_only=True).ip_addresses.first().address.ip }}/
```

**Reply by kotso** [reply]
@candlerb Thank you very much.

it worked, but correct for is `{{ obj.address.ip }}`
